chapters/CRUD-Operations.md

```diff
@@ -358,47 +434,50 @@ reached, or the blockchain history passes the supplied `targetTime`. At this
-* `didDocumentHistory`: An ordered array of DID documents from version 1 up to the
-   current version.
-* `updateHashHistory`: An ordered array of SHA256 hashes of ::DID Update Payloads:: 
+   resolution algorithm; REQUIRED; integer
+* `didDocumentHistory` - an ordered array of DID documents from version 1 up to the
+   current version; REQUIRED; array
+* `updateHashHistory` - an ordered array of SHA256 hashes of ::DID Update Payloads:: 
    that have been applied to the DID document by the resolution algorithm in order 
-   to construct the `contemporaryDIDDocument`.
-* `signalsMetadata`: A Map from Bitcoin transaction identifiers of ::Beacon Signals:: 
+   to construct the `contemporaryDIDDocument`; REQUIRED; array
+* `signalsMetadata` - a Map from Bitcoin transaction identifiers of ::Beacon Signals:: 
    to a struct containing ::Sidecar Data:: for that signal provided as part of the
-   resolutionOptions. This struct contains the following properties:
-   * `updatePayload`: A ::DID Update Payload:: which should match the update announced
+   `resolutionOptions`; REQUIRED; map containing the following properties:
+   * `updatePayload` - a ::DID Update Payload:: which SHOULD match the update announced
       by the ::Beacon Signal::. In the case of a ::SMT:: proof of non-inclusion no
-      DID Update Payload may be provided.
-   * `proofs`: A ::Sparse Merkle Tree:: proof that the provided `updatePayload` value is
-      the value at the leaf indexed by the **did:btc1** being resolved. TODO: What exactly this
+      DID Update Payload MAY be provided; OPTIONAL; object
+   * `proofs` - a ::Sparse Merkle Tree:: proof that the provided `updatePayload` value is
+      the value at the leaf indexed by the **did:btc1** being resolved. REQUIRED; TODO: What exactly this
       structure is needs to be defined.
-* `network`:  A string identifying the Bitcoin network of the **did:btc1** identifier.
-   The algorithm **MUST** query the Bitcoin blockchain identified by the `network`.
+* `network` -  a string identifying the Bitcoin network of the **did:btc1** identifier.
+   The algorithm MUST query the Bitcoin blockchain identified by the `network`; REQUIRED; string
 
 It returns the following output once either `targetTime` or `targetVersionId` have been reached:
-* `contemporaryDIDDocument`: A DID Core conformant DID document resolved using updates found
+
+* `contemporaryDIDDocument` - A ::contemporary DID document:: conformant to DID Core resolved using updates found
    on-chain from genesis to `contemporaryBlockheight`.
 
+The steps are as follows:
+
 1. Set `contemporaryHash` to the result of passing `contemporaryDIDDocument` into the 
-[JSON Canonicalization and Hash] algorithm.
-1. Find all `beacons` in `contemporaryDIDDocument`: All `service` in
-   `contemporaryDIDDocument.service` where `service.type` equals one of
-   `SingletonBeacon`, `CIDAggregateBeacon` and `SMTAggregateBeacon` ::Beacon::.
+   [JSON Canonicalization and Hash] algorithm.
```

Comment:
```suggestion
   [JSON Canonicalization and Hash] algorithm.
```

Wondering how we handle Ryans desire to move to RDFC. I am inclined to put a flag at the front of the spec. But leave the spec as is for now.

I do not think we are going to be able to get our libraries in sync and aligned using RDFC in time. Wheras today, they are in sync and working. I do not see a security concern with this personally. It is clearly a distinct crypto suite. And these are just going on the test network anyway. We are not version 1 yet. This is one of the areas we expect to change